### PR TITLE
Mobile and desktop version pull request

### PR DESCRIPTION
**Pull request by Muhammad-UsmanTariq**
### I complete this  mobile and desktop project using css and html according to the given conditions.

- [x]  Use media queries to make it responsive for desktop version.
- [x] Use grid and flexbox.
- [x] No use of javascript in this project.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -313,5 +389,606 @@ hr {
...
+
+button {
+  margin-bottom: 50px;
+}
+
+@media (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+  }
```
Comment (emhamza) [inline index.css:507]: @Muhammad-UsmanTariq I would suggest you to have look on your background img and make it similar to Figma design: 
osman style

**Review by emhamza — CHANGES_REQUESTED** [review]
Hi @Muhammad-UsmanTariq 🎉 

Good job so far 👍 There is still something that you need to update in order to go to the next project

tnnd-the-noise-next-door

File: index.css (hunk 1 above)
Comment (Muhammad-UsmanTariq) [reply index.css]: ### First my portfolio background is this

back

Comment (Muhammad-UsmanTariq) [reply index.css]: ### Now i fix it.

Screenshot 2023-08-04 184203

Comment (Muhammad-UsmanTariq) [reply index.css]: body {
    margin: 0;
    padding: 0;
  }
These lines are extar so i removed it from the code.

**Review by emhamza — APPROVED** [review]
Hi @Muhammad-UsmanTariq 💚 

Good job implementing the changes, there is nothing to say
**Your Project is Approved:**

 
appr

**Pull request merged**